ingestion/update_deploy.sh

```diff
@@ -1,23 +1,29 @@
 #!/bin/bash
 
-echo "API_URL: $API_URL"
-echo "API_WORKSPACE_OID: $API_WORKSPACE_OID"
+VERSION=1
+NAME=$(basename $0)
+AUTHOR="Iron Will Beard"
```

Comment:
Maybe a different name... :)

Reply:
Seriously.  If you want me to remove your name entirely and replace with...team name?  idk.  I'll be happy to.  Just say the word.